jaeger-core/src/main/java/com/uber/jaeger/samplers/ProbabilisticSampler.java

```diff
@@ -26,32 +26,38 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
-import lombok.ToString;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Value;
 
-@SuppressWarnings("EqualsHashCode")
-@ToString
+@Value
+@Getter(AccessLevel.NONE)
+@EqualsAndHashCode(of = {"samplingRate"})
 public class ProbabilisticSampler implements Sampler {
   public static final String TYPE = "probabilistic";
 
-  private final long positiveSamplingBoundary;
-  private final long negativeSamplingBoundary;
-  private final double samplingRate;
-  private final Map<String, Object> tags;
+  @Getter
+  double samplingRate;
+
+  long positiveSamplingBoundary;
+  long negativeSamplingBoundary;
+  Map<String, Object> tags;
 
   public ProbabilisticSampler(double samplingRate) {
     if (samplingRate < 0.0 || samplingRate > 1.0) {
-      throw new IllegalArgumentException(
-          "The sampling rate must be greater than 0.0 and less than 1.0");
+      throw new IllegalArgumentException("The sampling rate must be greater than 0.0 and less than 1.0");
     }
 
     this.samplingRate = samplingRate;
-    this.positiveSamplingBoundary = (long) (((1L << 63) - 1) * samplingRate);
-    this.negativeSamplingBoundary = (long) ((1L << 63) * samplingRate);
 
-    Map<String, Object> tags = new HashMap<>();
+    HashMap<String, Object> tags = new HashMap<>();
     tags.put(Constants.SAMPLER_TYPE_TAG_KEY, TYPE);
     tags.put(Constants.SAMPLER_PARAM_TAG_KEY, samplingRate);
     this.tags = Collections.unmodifiableMap(tags);
+
+    this.positiveSamplingBoundary = (long) (((1L << 63) - 1) * samplingRate);
+    this.negativeSamplingBoundary = (long) ((1L << 63) * samplingRate);
```

Comment:
I'll change them back, during the first iteration, they made sense, but now it looks like i'm simply moving things over for no reason.